### ISSUE TITLE
Let editors know when project initialization failed

### ISSUE THREAD

**Issue by jrieken** [OmniSharp/omnisharp-roslyn#164]
When loading csproj files and also project.json files there is potential for disaster - we might be using xbuild #163 or the user malformed the project.json. Current, OmniSharp prints warning and error messages to the console which isn't user friendly. So in addition we should use the emitter to send event down to the editor telling it about those problems.

**Comment by jrieken** [comment]
With some pull requests we now send an ```ErrorMessage``` when DNX or MSBuild have trouble. Also warnings and (handled) errors are reported
